network/mikrotik/snmp/mode/cpu.pm

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright 2018 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package network::mikrotik::snmp::mode::cpu;
```

Comment:
This is the standard Linux detailed CPU OIDs, please remove this mode as it is not needed.

You need to change the plugin.pm for Mikrotik file to points to linux cpu-detailed mode.

Reply:
My bad, please check cpu and cpu detailed mode in snmp_standard directory

Reply:
Yeah I'm a bit confused as this was the oid mikrotik suggested in their forums, I'll do some testing on our appliances and compare values with the ones in the actual interface.

This change will be on hold till I get the data.

Reply:
Stress test complete, both oids return the same value at the same time. Will remove this mode and use standard